src/ontology/uberon-edit.obo

```diff
@@ -224189,6 +224189,60 @@ intersection_of: part_of UBERON:0005610 ! iliac vein
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-06-17T14:34:50Z xsd:dateTime
 
+[Term]
+id: UBERON:8480001
+name: capillary of brain
+def: "A capillary that is part of the brain." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0001982 ! capillary
+intersection_of: part_of UBERON:0000955 ! brain
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-06-24T09:28:31Z xsd:dateTime
+
+[Term]
+id: UBERON:8480002
+name: thoracic aorta smooth muscle tissue
+def: "A portion of smooth muscle tissue that is part of the thoracic aorta." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0004178 ! aorta smooth muscle tissue
```

Comment:
This can just be smooth muscle tissue, instead of aorta smooth muscle tissue (tends to be better practice to let the reasoner do the work - not in this case but some times you might miss some things that reasoner would handle and all) - same applies to all your other terms

Reply:
OK, thanks. I will correct all the terms and send a pull request again